content/v0.75.0-rc1.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Introducing 0.75.0 Release Candidate 1"
+date = 2020-04-22
++++
+
+![Dark Forces (Win), Paku Paku (macOS), Settlers 2 (Linux)](/3games.png)
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+This release is significant for several reasons:
+
+1. Our first stable build - from this point forward, builds named
+   0.75.x will only contain bugfixes.
+
+2. Start of the new development cycle for the next release - we have a number
+   of planned features lined up already.
```

Comment:
It kicks off the 0.76 development cycle, for which a number of features are already lined up.